common/scala/src/main/resources/application.conf

```diff
@@ -12,3 +12,22 @@ akka.http {
         max-open-requests = 128
     }
 }
+
+akka.tracing {
+    #zipkin configuration
+    host = "localhost"
+    host = ${?TRACING_HOST} # Override default HOST by enviornment variable
+    port = 9410 # default
+    port = ${?TRACING_PORT} # Override default PORT by enviornment variable
+    sample-rate = 100 # by default trace every 100th request
+    sample-rate = ${?TRACING_SAMPLE_RATE} # Override default Sample_Rate by enviornment variable
+
+    enabled = false
+    enabled = ${?TRACING_ENABLED} # Override default Enabled by enviornment variable
```

Comment:
@adriancole this will override the hardcoded default by environment variable if its set [(https://www.playframework.com/documentation/2.5.x/ProductionConfiguration#Using-environment-variables)](https://www.playframework.com/documentation/2.5.x/ProductionConfiguration#Using-environment-variables)
Not sure if this is what your are referring to.